media/css/firefox/all.less

```diff
@@ -276,6 +276,7 @@ body.nightly {
     }
 
     #language-search {
+        width: @widthTablet - (@gridGutterWidth * 2 * 2.25);
```

Comment:
nah! That was not my intention. At tablet break-point (around 760-800 px) the width of `#language-search` is more than it's parents which is breaking the view. I couldn't figure out where the width is coming from so, thought about fixing the width by that wizardry :stuck_out_tongue_winking_eye: 

I haven't tested it on an actual devices. will attach a image when I get back home.